src/components/TagBox.js

```diff
@@ -53,10 +52,18 @@ class TagBox extends Component {
 
       tagBoxContents = (
         <div>
-          <p className="gif-name copy-icon" onMouseUp={ clip }>
-            <span className="sliding-ul">
-              <span className="lightgrey">{ ROOT_URL }</span>
-              { activeGif.src }
+          <input
+            type="text"
+            className="gif-fullname-hidden"
+            value={ ROOT_URL + activeGif.src }
```

Comment:
Just testing a new-style review for Squirrelbot

_this should be_

```
code
block
```

see a [link](https://github.com/bjacobel/gifs/pull/17/commits/fd584faa5a7f312f4373a301acac73762068c2a6#diff-560bfefb7832bf845febdb13e6f336feR66)